rebasehelper/options.py

```diff
@@ -184,6 +184,10 @@
         "name": ("--changelog-entry",),
         "default": "- New upstream release %{version}",
         "help": "text to use as changelog entry, can contain RPM macros, which will be expanded"
+    },
+    {
+        "name": ("--conf",),
+        "help": "path to config with preset arguments"
```

Comment:
What is `config with preset arguments`? Did you mean `predefined`? Anyway, I think `custom path to configuration file` or `load specified configuration file instead of a default` would be better.